subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/AnnotationProcessingCompileTask.java

```diff
@@ -162,7 +161,8 @@ private Processor decorateForIncrementalProcessing(Processor processor, Incremen
     }
 
     private Processor decorateForTimeTracking(Processor processor, AnnotationProcessorResult processorResult) {
-        return new TimeTrackingProcessor(processor, processorResult);
+        return processor;
+        //return new TimeTrackingProcessor(processor, processorResult);
```

Comment:
Is that a real change or a forgotten local modification?
Either way, needs to be cleaned up ;-)

Reply:
This is indeed not intended, good catch!